rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
-channel = "nightly-2022-03-10"
+channel = "nightly-2022-03-03"
```

Comment:
We need a rollback of toolchain?

Reply:
It looks work:
cargo update -p crossbeam-epoch:0.9.6 --precise 0.9.7